src/note-components/Sidebar/sidebar.jsx

```diff
@@ -0,0 +1,28 @@
+const Sidebar = () => {
+return (
+<div className="left-section">
+    <div className="list left-icon">
+        <span><i class="far fa-home text-color"></i></span>
+        <p>Home</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-tag text-color"></i></span>
+        <p>Label</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-archive text-color"></i></span>
+        <p>Archieve</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-trash text-color"></i></span>
+        <p>Trash</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-user-circle text-color"></i></span>
+        <p>Profile</p>
+    </div>
```

Comment:
Can have a list of nav links which you can map over
```js
Can have a list of colors which you can map over

[n1,n2,n3,n4,n5],map((everyLink,index)=>{
return (
<div key={everyLink} className=`list left-icon`>
<p>{everyLink}</p>
</div>
)
})
```

Reply:
Okay